src/control/server/mgmt_pool_test.go

```diff
@@ -132,6 +133,108 @@ func TestServer_MgmtSvc_PoolCreateAlreadyExists(t *testing.T) {
 	}
 }
 
+func TestServer_MgmtSvc_calculateCreateStorage(t *testing.T) {
+	defaultTotal := uint64(10 * humanize.TByte)
+	defaultRatio := 0.06
+	defaultScmBytes := uint64(float64(defaultTotal) * defaultRatio)
+	defaultNvmeBytes := defaultTotal
+	testTargetCount := 8
+	scmTooSmallRatio := 0.01
+	scmTooSmallTotal := uint64(testTargetCount * ioserver.NvmeMinBytesPerTarget)
+	scmToolSmallReq := uint64(float64(scmTooSmallTotal) * scmTooSmallRatio)
```

Comment:
nit - spelling
```suggestion
	scmTooSmallReq := uint64(float64(scmTooSmallTotal) * scmTooSmallRatio)
```

Reply:
Gah, fixed that in one repo and forgot to bring it over to the one I pushed from. Fixed.